sdk/apis/tenancy/v1alpha1/types_workspace.go

```diff
@@ -174,6 +174,50 @@ type WorkspaceSpec struct {
 	//
 	// +kubebuilder:format:uri
 	URL string `json:"URL,omitempty"`
+
+	// Mount is a reference to a an object implementing a mounting feature. It is used to orchestrate
+	// where the traffic, intended for the workspace, is sent.
+	// If specified, logicalcluster will not be created and the workspace will be mounted
+	// using reference mount object.
+	//
+	// +optional
+	Mount *Mount `json:"mount,omitempty"`
```

Comment:
It's immutable. It must be doc'ed and checked. Is it?

Reply:
updated.